### PR TITLE
UG: update cookRecipe

### LINKED ISSUE #180: [PE-D] Unclear instructions on terminal
Mentions insufficient ingredients even with quantity:3

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: harithadiv/ped#7

### PR DESCRIPTION
**Pull request by alwayshuizhen**
The following have been added:
- more examples for `addrecipe`
- more explanations for `cookrecipe` and its functionality is shown with examples

This PR closes #180

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -206,12 +206,14 @@ The different types of `CATEGORY` are listed below:
   
 Any `CATEGORY` that does not falls in the list could be put under `Miscellaneous`.
 
-All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#332-list-recipes-listrecipe-1)  
+> All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#332-list-recipes-listrecipe-1)  
+> Please note that `RECIPE_NAME` and `INGREGIENT_NAME` can contain spaces. These will not be removed after addition. (i.e. "____Chicken Stew" where `_` is space will remain )
```
Comment (zi-hui) [inline docs/UserGuide.md]: Good job on updating the UG but there is a typo for INGREGIENT_NAME, ingredient is spelled wrongly.

Comment (alwayshuizhen) [reply docs/UserGuide.md]: Thanks for the catch! 😄

File: docs/UserGuide.md [hunk 2]
```diff
@@ -213,12 +213,14 @@ The different types of `CATEGORY` are listed below:
   
 Any `CATEGORY` that does not falls in the list could be put under `Miscellaneous`.
 
-All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#342-list-recipes-listrecipe)  
+> All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#332-list-ingredient-listingredient)
```
Comment (jinfayap) [inline docs/UserGuide.md]: @alwayshuizhen `listrecipe all` linking to `listrecipe header` or `listingredient header`?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -853,21 +853,37 @@ Use case ends.
...
+#### F.5. Cook a recipe
+1. Cooks the specified recipe and ingredients in the recipe will be automatically deducted.
+    1. Prerequisites: List all the ingredient using the `listingredient all` command.
+    1. Test case (sufficient ingredient): `cookrecipe /n warm milk /p 2`\
+    Expected: A reduction of the ingredients' quantity multiplied by `2` can be noticed when listing the ingredients with `listingredient all`
+    1. Test case (Sufficient even with expired ingredients):  `cookrecipe /n warm milk /p 2`\
+    Expected: The automatic deduction will not be carried out and expired item will be notified to user.
+    1. Test case: (Insufficient even with expired ingredients): `cookrecipe /n warm milk /p 2`\
+    Expected: The automatic deduction will not be carried out.
+
```
Comment (jinfayap) [inline docs/DeveloperGuide.md:873]: @alwayshuizhen , Good job adding this. Could you rectify the content page list in the `appendix F` also? Does not correspond when you added all these in.

**Review by jinfayap — CHANGES_REQUESTED** [review]

**Review by jinfayap — APPROVED** [review]
Fixes are completed. LGTM 👍

**Pull request merged**